cypress/integration/inventory/export.spec.js

```diff
@@ -0,0 +1,40 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryActions from '../../support/fragments/inventory/inventoryActions';
+import FileManager from '../../support/utils/fileManager';
+
+
+describe('inventory / actions: export UUIDs', () => {
+  beforeEach('navigates to Inventory and delete downloads folder', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C9284 verifies export UUIDs instances', () => {
+    cy.do([
+      InventorySearch.searchByEffectiveLocation(),
+      InventoryActions.open(),
```

Comment:
and 3000 can be moved into env variables

Reply:
> additionally, can we put this code into Inventories.saveInstancesUUIDs():
" cy.wait('@getids').then((obj) => {
const expectedUUIDs = [];
obj.response.body.ids.forEach((elem) => { expectedUUIDs.push(elem.id); });
// Need time for download file TODO: think about how it can be fixed
  cy.wait(3000);"
expectedUUIDs can be return value, which will be processed in next steps of this test

I tried but I couldn't put working with interceptor into separate function. It borns a lof of exception and we need to think about asyn and sync code with this solution. But if I use interseptor in test it so easy works without exceptions